Formula/dynare.rb

```diff
@@ -24,14 +24,14 @@ class Dynare < Formula
   depends_on "boost" => :build
   depends_on "cweb" => :build
   depends_on "fftw"
-  depends_on "gcc" # for gfortran
```

Comment:
Please leave the `gcc` comment unchanged everywhere. It's identical in all formulas that rely on GCC to get gfortran, which makes it easy to grep for when GCC is updated

Reply:
sure thing. IMO, it would have been nice to just say "Fortran" but, if it's already in place, let's not worry about updating it

Reply:
reverted all the comments. Still an issue building Octave due to java. I'm not sure how to fix it. Let's see what CI says once it gets a chance to go through.